package.json

```diff
@@ -2,7 +2,7 @@
   "name": "gather",
   "displayName": "Gather",
   "description": "Adds gather functionality to the Python Extension",
-  "version": "2020.7.1",
+  "version": "2020.11.0",
   "publisher": "ms-python",
```

Comment:
Publisher to ms-toolsai?

Reply:
I'm going to have to do this later, if I just change the publisher it won't work. I need to register the new name with Kai.
Here's another issue tracking that: https://github.com/microsoft/vscode-jupyter/issues/416